From 23d87c92b7211cc5249eefe4ab3a2c7f370e279e Mon Sep 17 00:00:00 2001
From: Zack Rusin <zackr@vmware.com>
Date: Thu, 14 Feb 2013 22:33:01 +0000
Subject: Kill mibstore

It was a noop for at least 5 years and it has been removed.

Signed-off-by: Zack Rusin <zackr@vmware.com>
---
diff --git a/saa/saa_priv.h b/saa/saa_priv.h
index 6652054..5d81453 100644
--- a/saa/saa_priv.h
+++ b/saa/saa_priv.h
@@ -48,7 +48,6 @@
 #include "pixmapstr.h"
 #include "windowstr.h"
 #include "servermd.h"
-#include "mibstore.h"
 #include "colormapst.h"
 #include "gcstruct.h"
 #include "input.h"
diff --git a/src/vmware.c b/src/vmware.c
index 41201f1..6b74e14 100644
--- a/src/vmware.c
+++ b/src/vmware.c
@@ -23,7 +23,6 @@ char rcsId_vmware[] =
 #include "xf86Pci.h"		/* pci */
 
 #include "mipointer.h"		/* sw cursor */
-#include "mibstore.h"		/* backing store */
 #include "micmap.h"		/* mi color map */
 #include "vgaHW.h"		/* VGA hardware */
 #include "fb.h"
@@ -1483,7 +1482,6 @@ VMWAREScreenInit(SCREEN_INIT_ARGS_DECL)
      * If backing store is to be supported (as is usually the case),
      * initialise it.
      */
-    miInitializeBackingStore(pScreen);
     xf86SetBackingStore(pScreen);
     xf86SetSilkenMouse(pScreen);
 
diff --git a/vmwgfx/vmwgfx_driver.c b/vmwgfx/vmwgfx_driver.c
index 9959ffb..7863ba2 100644
--- a/vmwgfx/vmwgfx_driver.c
+++ b/vmwgfx/vmwgfx_driver.c
@@ -1036,7 +1036,6 @@ drv_screen_init(SCREEN_INIT_ARGS_DECL)
 		   (ms->only_hw_presents) ? "enabled" : "disabled");
     }
 
-    miInitializeBackingStore(pScreen);
     xf86SetBackingStore(pScreen);
     xf86SetSilkenMouse(pScreen);
     miDCInitialize(pScreen, xf86GetPointerScreenFuncs());
--
cgit v0.9.0.2-2-gbebe
